tools/sggolangcilint/tools.go

```diff
@@ -33,15 +33,32 @@ func defaultConfigPath() string {
 }
 
 func CommandInDirectory(ctx context.Context, directory string, args ...string) *exec.Cmd {
-	configPath := filepath.Join(directory, ".golangci.yml")
-	if _, err := os.Lstat(configPath); errors.Is(err, os.ErrNotExist) {
-		configPath = defaultConfigPath()
+	cmdArgs := []string{"run"}
+
+	skipConfig := false
+	for _, arg := range args {
+		if arg == "--no-config" {
```

Comment:
not important to do now, but let's bump to Go 1.22 and use slices.Contains? :smile_cat:

Reply:
then you might even inline the whole variable and just say `if slices.Contains(...)`

Reply:
We have panics with 1.22  in book-service, so that requires investigations. I would rather do that separately.